pyproject.toml

```diff
@@ -14,7 +14,7 @@ python_version = "3.8"
 
 [tool.ruff]
 extend-exclude = ["thirdparty"]
-extend-include = ["SConstruct", "SCsub"]
+extend-include = ["*SConstruct", "*SCsub"]
```

Comment:
Before:
```
$ ruff format
106 files left unchanged.
```
After:
```
277 files left unchanged.
```

(We have a couple nested `SConstruct`s too in the TextServer modules for their GDExtension based builds.)